docs/api/vulnerability-info.md

```diff
@@ -0,0 +1,114 @@
+
+# Vulnerability information
+
+NuGet client, starting from version 6.7, can download known package vulnerability information to use in scenarios such as checking packages during restore operations.
+
+The API consists of at least two files, the [vulnerability index](#vulnerability-index) and one or more [vulnerability data files](#vulnerability-data).
+Known vulnerability data can be partitioned into multiple files, and the vulnerability index provides clients with information needed to cache files, and update the cache, efficiently.
+
+The resource used for building this URL is the `VulnerabilityInfo` resource found in the [service index](service-index.md).
+
+## Versioning
+
+The following `@type` values are used:
+
+@type value                       | Notes
+--------------------------------- | -----
+VulnerabilityInfo/6.7.0 | The initial release
+
+## Suggested partitioning strategy
+
+The files listed in the vulnerability index should ideally be optimized to maximize caching, and therefore minimize updates to large files.
+This will allow clients to minimize the frequency it needs to download updates.
+
+A suggested strategy for vulnerability data partitioning is to have two files, `base.json` and `updates.json`.
+The `base.json` file is updated periodically (for example once a month), and contains all known vulnerabilities at the time the file is regenerated.
+The `updates.json` file should any new advisories published since `base.json` was last regenerated.
+This will allow clients to download the large `base.json` infrequently, while the frequently changing `updates.json` file is always relatively small.
+
+NuGet client does not provide a way for one file to update or remove entries from another file, so if any modifications are needed to any advisory in `base.json`, then `base.json` can be regenerated and all entries in `updates.json` merged.
+
+## Vulnerability index
+
+The vulnerability index is a JSON array of objects with the following properties:
+
+Name     | Type     | Required | Notes
+-------- | ---------| -------- | -----
+@name    | string   | yes      | A short name for the file, used as a cache key.
+@id      | string   | yes      | The full (absolute) URL to a vulnerability data file.
+@updated | DateTime | yes      | An ISO 8601 string representing the date and time the file was last updated, ideally with the UTC timezone.
+comment  | string   | no       | A optional descriptive string.
+
+The following restrictions apply:
+
+* The index must be an array of objects with between 1 and 16 items.
+* `@name` must be unique within the index, must be between 1 and 32 characters long, and can only use the characters `A` to `Z`, `a` to `z`, `0` to `9`, `-`, or `_`.
+* `@id` must be an absolute URL, not a relative URL.
+
+## Vulnerability data
+
+Vulnerability data files are a JSON object used as a dictionary.
+Property keys are package identifiers and values are an array of the following object with the following properties:
```

Comment:
Using the term `identifier` is inconsistent with the rest of the docs. I recommend `package ID`.